apps/expo/tailwind.config.ts

```diff
@@ -5,4 +5,11 @@ import baseConfig from "@cfd/tailwind-config";
 export default {
   content: ["./src/**/*.{ts,tsx}"],
   presets: [baseConfig],
+  theme: {
+    extend: {
+      colors: {
+        checkGreen: "#22AB00",
+      },
+    },
+  },
```

Comment:
wouldn't recommend putting this in theme, as we'll be adding the actual theme values from our design system soon

Reply:
That makes sense!